README.md

```diff
@@ -120,8 +120,9 @@ You can cache the installed gems with these two steps:
         bundle install --jobs 4 --retry 3
 ```
 
-When using a single job with a Ruby version, replace `${{ matrix.ruby }}` with the Ruby version.  
-When using `.ruby-version`, replace `${{ matrix.ruby }}` with `${{ hashFiles('.ruby-version') }}`.  
+When using a single os, replace `${{ matrix.os }}` with `${{ runner.os }}`.
+When using a single job with a Ruby version, replace `${{ matrix.ruby }}` with the Ruby version.
```

Comment:
Could you keep the trailing spaces here and below? They force a newline in rendered Markdown.